client/src/styles/LeftAndMain_Subsites.scss

```diff
@@ -75,9 +69,7 @@
   .field {
     margin-left: 100px;
   }
-}
 
-#Form_AddSubsiteForm {
```

Comment:
Why is this ID being removed?

Reply:
It's not being removed - this block with a duplicate selector is being merged in with the above block which has the exact same selector. The dist result is identical.